iroh-net/src/magicsock.rs

```diff
@@ -287,6 +287,9 @@ impl MagicSock {
     ///
     /// [`Callbacks::on_endpoint`]: crate::magicsock::conn::Callbacks::on_endpoints
     pub async fn new(opts: Options) -> Result<Self> {
+        #[cfg(features = "derp-only")]
+        debug!("creating MagicSock that will only send packets over a derp relay connection.");
```

Comment:
why not make this `warn!`?  i mean, debug is fine too, but more prominent wouldn't hurt

Reply:
great idea